capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -0,0 +1,140 @@
+const inp = document.getElementById('search');
+const currentFocus = -1;
+/**
+ * @param {Object} autocompleteListElmt
```

Comment:
Have look at [section 7](https://google.github.io/styleguide/jsguide.html#jsdoc) in the JSDoc style guide and see if you can bring the JSDoc in this file more in line with the style guide...

Reply:
Looked at the section 7 documentation & updated a lot of the JSDoc-let me know if you expected more to change!